README.md

```diff
@@ -179,6 +179,33 @@ depicting instructional materials -- textbooks etc on the topic):
     [9] Fisher, R.A., 1936. The use of multiple measurements in taxonomic problems. Annals of eugenics, 7(2), pp.179–188.
     [10] Gower, J.C. & Ross, G., 1969. Minimum spanning trees and single linkage cluster analysis. Applied statistics, pp.54–64.
     [11] Sibson, R., 1973. SLINK: an optimally efficient algorithm for the single-link cluster method. The Computer Journal, 16(1), pp.30–34.
+    
+The `DUECREDIT_REPORT_ALL` flag allows one to output all the references
+that were available but not directly used in the script. Compared to 
```

Comment:
A little incorrectly stated... it is not all the references (more stuff had references), but references for the modules which didn't have any citeable functionality used